src/style.css

```diff
@@ -0,0 +1,83 @@
+body {
+  margin: 0;
+  --height: 300px;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+  min-width: var(--height);
+  line-height: var(--height);
```

Comment:
What does fit better – `height` or `line-height`?

Reply:
i think {line-heght} - it helps to center text vertical